Makes the .md linter stop screaming.

TemporalOroboros opened the pull request:
The .md linter was complaining.

MichalYoung commented:
I'm looking at the diff, and to me it looks like replacing lines with identical lines.  Can you describe briefly what the problem was and how you fixed it?

TemporalOroboros commented:
The linter on the program I was using does not approve of spaces at the end of lines and has some pretty stringent rules on how many empty lines can exist between sections (one). I went through and deleted or added whitespace as needed until it stopped painting over the minimap with yellow highlights. A lot of the identical lines are just within thinking that because I added a new empty line before/after a paragraph the whole paragraph is brand new.
